src/operator/channel_op_common.h

```diff
@@ -101,6 +101,42 @@ void Split(const mshadow::Tensor<xpu, dim, DType> &input,
     split_helper<xpu, dim, dim-1>(input, output, dimension, req);
   }
 }
+
+template<typename cpu, int dim, typename DType>
+void Split_2D(const mshadow::Tensor<cpu, dim, DType> &input,
+           std::vector<mshadow::Tensor<cpu, dim, DType> > *output,
+           const int dimension, const std::vector<OpReqType> &req) {
+  if (dimension != 1) {
```

Comment:
Same coding style as  functions in this file did such as Split(), Concatenate()

Reply:
`split()` and `concatenate()` uses `if` branch because there are multiple conditions to check. It would be more appropriate to use `CHECK()` here as Zhennan suggested.